lib/Core/Executor.cpp

```diff
@@ -4136,13 +4136,15 @@ void Executor::runFunctionAsMain(Function *f,
 
   // hack to clear memory objects
   delete memory;
-  memory = new MemoryManager(NULL);
+  //memory = new MemoryManager(NULL);
 
   globalObjects.clear();
   globalAddresses.clear();
 
   if (statsTracker)
     statsTracker->done();
+
+	tries++;
```

Comment:
if we move `bindModuleConstants()` to KLEE's main.cpp, we may be able to get rid of the `tries` variable.